closure/templates/BUILD

```diff
@@ -39,7 +39,6 @@ closure_js_library(
     name = "soyutils_usegoog",
     srcs = ["@soyutils_usegoog//file"],
     suppress = [
-        "JSC_MUST_HAVE_TRAILING_UNDERSCORE",
```

Comment:
This feature you added is amazing. I certainly would have missed removing it without the error.